dbux-projects/src/projectLib/Project.js

```diff
@@ -413,7 +428,7 @@ This may be solved by using \`Delete project folder\` button.`);
 
     // NOTE: returns status code 1, if there are any changes, IFF --exit-code or --quiet is provided
     // see: https://stackoverflow.com/questions/28296130/what-does-this-git-diff-index-quiet-head-mean
-    const code = await this.exec('git diff-index --exit-code HEAD --', { failOnStatusCode: false });
+    const code = await this.exec('git diff-index --exit-code HEAD --', { failOnStatusCode: false, logStdout: false });
```

Comment:
Again, we want to see the `stdout`. If Dbux any has any amount of users, I can foresee many complaints relating to exactly this feature. We will need a lot of info, to make any sense of the problem and help anyone.